requirements/base.txt

```diff
@@ -6,4 +6,5 @@ google-cloud-storage==1.26.0
 better-setuptools-git-version==1.0.5
 unittest-xml-reporting==3.0.2
 wheel==0.35.1
-setuptools==49.6.0
\ No newline at end of file
+setuptools==49.6.0
+google-cloud-logging==1.15.1
```

Comment:
I think that logging should be optional (just as monitoring, bigquery support, etc.)